zpi-frontend/src/pages/UploadingFiles/UploadEmployeeFile.tsx

```diff
@@ -143,12 +158,68 @@ function UplaodEmployeeFilePage() {
       const formData = new FormData();
       formData.append('file', file);
 
-      api.post('http://localhost:8080/file/employee', formData)
+      axios
```

Comment:
tu ma być api.post ...itd bez tego headers bo to jest ustawiane w utilsach, więc to trzeba by zmienić